### ISSUE TITLE
[TRG 5.09] Enhance Helm test workflow

### ISSUE THREAD

**Issue by SebastianBezold** [eclipse-tractusx/bpdm#403]
## TL;DR

- Add a `workflow_dispatch` trigger to the helm test workflow
- Add an input variable for [kindest/node image](https://hub.docker.com/r/kindest/node), so test can be run on different kubernetes versions 

## Description 

The current [helm-chart-lint.yaml](https://github.com/eclipse-tractusx/bpdm/blob/main/.github/workflows/helm-chart-lint.yaml) workflow is only triggered by automated events (PRs and pushes). 

To make quality gate reviews easier, there should also be way to trigger the run on-demand (`workflow_dispatch`). The on-demand run should test the same things, that the existing helm test workflow is already testing. Additionally, there should be a possibility to test on different kubernetes versions. This can be achieved, by specifying the [kindest/node](https://hub.docker.com/r/kindest/node/tags) image, that is used by the kind action

## Additional info

If combining `workflow_dispatch` with input variables and automated triggers does not work well in a single workflow, separate workflows could be chosen as a more suitable approach

**Comment by nicoprow** [comment]
Has been now implemented as part of the another issue #645
